### DISCUSSION TITLE
Uninstalling Security Onion 2 on Ubuntu 18.04

### DISCUSSION THREAD

**Discussion by RobertUH** [Security-Onion-Solutions/securityonion#3138] [Unsupported Versions]
Is there any way to completely uninstall all SO2 packages / logs? I ran into disk issues quickly due to a poor config on my part and I'm trying to wipe the system clean without having to reinstall Ubuntu. I'm on a standalone deployment for reference.

**Comment by weslambert** [comment]
There is no way at the moment for a complete wipe without reinstalling via ISO, etc.  You should be able to re-run setup without too much of an issue.
